spec

```diff
@@ -615,7 +615,6 @@ endseg
 beginseg
     name "ovl_title"
     include "build/src/overlays/gamestates/ovl_title/z_title.o"
-    include "build/data/overlays/gamestates/z_title.data.o"
     include "build/data/overlays/gamestates/z_title.reloc.o"
```

Comment:
If the overlay matches you should change the spec to use the auto generated _reloc file and remove the reloc.s file.